cmd/erasure-sets.go

```diff
@@ -1326,21 +1339,14 @@ func (s *erasureSets) HealBucket(ctx context.Context, bucket string, opts madmin
 	}
 
 	for _, set := range s.sets {
-		var healResult madmin.HealResultItem
-		healResult, err = set.HealBucket(ctx, bucket, opts)
+		healResult, err := set.HealBucket(ctx, bucket, opts)
 		if err != nil {
 			return result, toObjectErr(err, bucket)
 		}
 		result.Before.Drives = append(result.Before.Drives, healResult.Before.Drives...)
 		result.After.Drives = append(result.After.Drives, healResult.After.Drives...)
 	}
 
-	// Check if we had quorum to write, if not return an appropriate error.
-	_, afterDriveOnline := result.GetOnlineCounts()
-	if afterDriveOnline < ((s.setCount*s.setDriveCount)/2)+1 {
-		return result, toObjectErr(errErasureWriteQuorum, bucket)
-	}
-
```

Comment:
this is not a correct assessment removed it no need to proactively return a write quorum error.